sql/hive/src/main/scala/org/apache/spark/sql/hive/orc/OrcFileOperator.scala

```diff
@@ -125,16 +125,4 @@ private[hive] object OrcFileOperator extends Logging {
       path: String, conf: Option[Configuration]): Option[StructObjectInspector] = {
     getFileReader(path, conf).map(_.getObjectInspector.asInstanceOf[StructObjectInspector])
   }
-
-  def listOrcFiles(pathStr: String, conf: Configuration): Seq[Path] = {
```

Comment:
Removed this method because duplicate of 
https://github.com/apache/spark/blob/99213d7123a88e1fc79bc73b898c4971c78f37ff/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcUtils.scala#L64

Reply:
For the dev API, please don't delete it.

Reply:
Alright. What the process?

Reply:
Sorry was too short. What the process of deleting this kind of duplicates?